docs/advanced-ai/intro-tutorial.md

```diff
@@ -1,217 +1,127 @@
-
-## Step seven: Test the workflow
-
-You can now try out the workflow: close any open nodes, then select **Chat** to start talking with the AI. Ask `Please give me a list of fruits`. The AI Agent node executes the workflow you specified in **Workflow ID**, passing in the value of `visibility` from the **Workflow Values** field. The workflow generates some sample data, filters it to remove any items that aren't set to `public`, and returns the resulting data.
+## 2. Add a trigger node
+
+Every workflow needs somewhere to start. In n8n these are called 'trigger nodes'. For this workflow, we want to start with a chat node.
+
+ 1. Select **Add first step** or press ++Tab++ to open the node menu.
+
+ 1. Search for **Chat Trigger**. n8n shows a list of nodes that match the search.
+
+ 1. Select **Chat Trigger** to add the node to the canvas. n8n opens the node.
+
+ 1. Close the node details view (Select **Back to canvas**) to return to the canvas.
+
+??? explanation "Explanation..."
+    The trigger node generates output when there is an event causing it to trigger. In this case we want to be able to type in text to cause the workflow to run. In production, this trigger can be hooked up to a public chat interface as provided by n8n or embedded into another website. To start this simple workflow we will just use the built-in local chat interface to communicate, so no further setup is required.
+
+<n8n-demo workflow="%7B%0A%20%20%22name%22%3A%20%22AI%20tutorial%22%2C%0A%20%20%22nodes%22%3A%20%5B%0A%20%20%20%20%7B%0A%20%20%20%20%20%20%22parameters%22%3A%20%7B%0A%20%20%20%20%20%20%20%20%22options%22%3A%20%7B%7D%0A%20%20%20%20%20%20%7D%2C%0A%20%20%20%20%20%20%22type%22%3A%20%22%40n8n%2Fn8n-nodes-langchain.chatTrigger%22%2C%0A%20%20%20%20%20%20%22typeVersion%22%3A%201.1%2C%0A%20%20%20%20%20%20%22position%22%3A%20%5B%0A%20%20%20%20%20%20%20%20-200%2C%0A%20%20%20%20%20%20%20%20-40%0A%20%20%20%20%20%20%5D%2C%0A%20%20%20%20%20%20%22id%22%3A%20%22a2d42e1f-36df-4a6a-a3b4-99a162074d11%22%2C%0A%20%20%20%20%20%20%22name%22%3A%20%22When%20chat%20message%20received%22%2C%0A%20%20%20%20%20%20%22webhookId%22%3A%20%2297c1a41f-8ef0-4d63-a924-92eb634384d3%22%0A%20%20%20%20%7D%0A%20%20%5D%2C%0A%20%20%22pinData%22%3A%20%7B%7D%2C%0A%20%20%22connections%22%3A%20%7B%7D%2C%0A%20%20%22active%22%3A%20false%2C%0A%20%20%22settings%22%3A%20%7B%0A%20%20%20%20%22executionOrder%22%3A%20%22v1%22%0A%20%20%7D%2C%0A%20%20%22versionId%22%3A%20%22b1641385-c6b0-48a8-8e26-20d1f6bd7fda%22%2C%0A%20%20%22meta%22%3A%20%7B%0A%20%20%20%20%22instanceId%22%3A%20%22cb484ba7b742928a2048bf8829668bed5b5ad9787579adea888f05980292a4a7%22%0A%20%20%7D%2C%0A%20%20%22id%22%3A%20%22l05TkWXXYbOiuL4o%22%2C%0A%20%20%22tags%22%3A%20%5B%5D%0A%7D"></n8n-demo>
```

Comment:
seemed more useful to establish a pattern of using our embedded workflow styles for these. I have a slight niggle with scrolling (if you are scrolling the text and you happen to land on one of these, you proceed to scroll the workflow rather than the text) and maybe these ones are simple enough that screenshots would be fine?
The main advantage here I see is that automating the generation of these parts could be fairly trivial and not involve filling up the repo with more images

Reply:
I think you're right about using the workflow component to render these. That might not have been an option when this was initially written.

I might not have understood your explanation above fully, but is there a reason you decided to define these in this way? Here, you're referencing the `n8n-demo` component directly, percent encoding the workflow, and define it inline. The way we typically do this is to put the workflow file in the `docs/_workflows` directory and then use the `workflowDemo` function to reference it [like this](https://github.com/n8n-io/n8n-docs/blob/4ef2977ba49c7ba02c5b4159f44eb23be453fd3d/docs/integrations/builtin/core-nodes/n8n-nodes-base.form.md?plain=1#L12). Are there advantages to doing it the way you do here?

Reply:
Ah, no particular reason. I didn't find any appropriate examples. I guess the only advantage of embedding everything is that it makes it easier to test ( a subject for another day), but I will redo these for now then and maybe update the wiki to include it

Reply:
Hmmm... is the standard here that it opens the code in a new tab? I think I prefer the forum style where it reveals the code block under the diagram and has a neat copy button. Out of scope for this work but maybe we should discuss that